webrtc-stats.html

```diff
@@ -532,12 +510,16 @@ <h3>
           </dt>
           <dd>
             <p>
-              Contains statistics related to a specific MediaStreamTrack and the corresponding
-              media-level metrics. It is accessed by either
+              Contains statistics related to a specific MediaStreamTrack's connection to this
+              PeerConnection and the corresponding media-level metrics. It is accessed by either
               <code><a>RTCLocalVideoStreamTrackStats</a></code> or
               <code><a>RTCLocalAudioStreamTrackStats</a></code>, both inherited from
               <code><a>RTCMediaStreamTrackStats</a></code>.
             </p>
+            <p>
+              The monitored "track" object is deleted when the track is disconnected from the
+              PeerConnection.
```

Comment:
When exactly is that? Is it in the in-parallel steps of [replaceTrack](http://w3c.github.io/webrtc-pc/#dom-rtcrtpsender-replacetrack)?

I'd love to nail down when the event fires. Could we say it MUST fire before replaceTrack's promise is resolved?

Reply:
Changed to "when the sender has its track value changed". Probably best to incorporate this in webrtc-pc algorithm too. Also updated the definition - the "this is an attachment" language hadn't made its way down here.